l10n_uy_edi/models/l10n_uy_cfe.py

```diff
@@ -1010,7 +1008,7 @@ def _uy_cfe_B8_DscItem(self, line):
         if line.product_id.l10n_uy_additional_info:
             res.append(line.product_id.l10n_uy_additional_info)
         if self._is_uy_inv_type_cfe():
-            if line.product_id.display_name != line.name:
+            if line.name and line.product_id and line.name != line.product_id.display_name:
                 res.append(line.name)
```

Comment:
En el caso de que nosotros quisieramos colocar una linea sin product_id esto no funcionaria.
```suggestion
            if line.name and line.product_id and line.name != line.product_id.display_name or line.name and not line.product_Id:
                res.append(line.name)
```